src/test/java/net/sf/jsqlparser/statement/alter/AlterTest.java

```diff
@@ -146,6 +146,14 @@ public void testAlterTableDropConstraint() throws JSQLParserException {
         assertEquals(alterExpression.getConstraintName(), "YYY");
     }
 
+    @Test
+    public void testAlterTableDropConstraintIfExists() throws JSQLParserException {
+        Statement stmt = CCJSqlParserUtil.parse("ALTER TABLE Persons DROP CONSTRAINT IF EXISTS UC_Person");
+
+        // not yet interesting enough to remember 'IF EXISTS' flag on constraint
```

Comment:
The deparser should handle this. Therefore it should remember it.

Reply:
So you want an ifExists-flag on the constraint as well?